codec/reflectcodec/type_codec.go

```diff
@@ -91,6 +91,13 @@ func (c *genericCodec) Size(value interface{}) (int, error) {
 
 // size returns the size of the value along with whether the value is constant sized.
 func (c *genericCodec) size(value reflect.Value) (int, bool, error) {
+	return c.sizeWithOmitEmpty(value, false)
+}
+
+// size returns the size of the value along with whether the value is constant
```

Comment:
```suggestion
// sizeWithOmitEmpty returns the size of the value along with whether the value is constant
```
should we rename this to `sizeWithNullable`?